scripts/qa_crowbarsetup.sh

```diff
@@ -3572,8 +3572,10 @@ function oncontroller_magnum_service_setup
     # not be fixed, we are going to have our own integration tests with SLES Magnum image
     local service_image_name=magnum-service-image
     local service_image_url=http://clouddata.cloud.suse.de/images/$arch/other/${service_image_name}.qcow2
-    local service_sles_image_name=sles-openstack-magnum-kubernetes
-    local service_sles_image_url=http://download.suse.de/ibs/Devel:/Docker:/Images:/SLE12SP1-JeOS-k8s-magnum/images/${service_sles_image_name}.${arch}.qcow2
+
+    local service_sles_image_name=openstack-magnum-k8s-image
+    local service_sles_image_path=$(find /srv/tftpboot/files/${service_sles_image_name} \
```

Comment:
we should download the file from the admin node instead of expecting it to be on the controller. see https://trello.com/c/QOkOA9XT